src/getConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { exec } from 'child_process';
+
+interface Config {
+  project: {
+    android: {
+      packageName: string;
+    }
+  }
```

Comment:
```suggestion
    };
  };
```

Otherwise prettier is complaining 🙃